.editorconfig

```diff
@@ -0,0 +1,104 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = tab
```

Comment:
il faudra passer ça en `indent_style = space` sinon il y a une incohérence entre ce qu'on voit sur Github et ce qu'on voit dans Android Studio, l'IDE est perdu ^^

Reply:
In fact it let you define the space you want when you use tabs. I can configure my IDE to tell a tab looks like 8 spaces for example. I would let it like that 🙏🏻

Reply:
It should be 4 spaces for Android project I think your current setting is 2 spaces?